packages/shared/src/chunker.ts

```diff
@@ -1,16 +1,25 @@
 // No clue why, but 3600 matches 4kb in the browser
-const MAX_CHUNK_SIZE = 3600;
-const re = new RegExp('.{1,' + MAX_CHUNK_SIZE + '}', 'g');
+let MAX_CHUNK_SIZE = 3600;
```

Comment:
I'd rewrite this whole thing to:

```js
const MAX_CHUNK_SIZE = 3600;
const MAX_CHUNK_REGEXP = createChunkRegExp(MAX_CHUNK_SIZE);

function createChunkRegExp(chunkSize: number) {
  return new RegExp('.{1,' + MAX_CHUNK_SIZE + '}', 'g');
}

export function createChunks(key: string, value: string, chunkSize?: number): Chunk[] {
  const re = chunkSize !== undefined ? createChunkRegExp(chunkSize) : MAX_CHUNK_REGEXP;
  
  // ...
}
```

It preserves a constant default, no type checking, and allows for optional overrides for tests.
It also doesn't export unnecessary, test-only functions.